onchain/Scarb.toml

```
@@ -6,9 +6,18 @@ version = "0.1.0"
 starknet = "2.6.0"
 openzeppelin = { git = "https://github.com/openzeppelin/cairo-contracts", branch = "main" }
 
+[dev-dependencies]
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry", tag = "v0.22.0" }
```

use the latest release
```suggestion
snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry", tag = "v0.25.0" }
```

---

Yes, 0.25.0 is stable, am using it now, i will push some changes i made that pass the first 11 test with snforge